config/configs_basic.yaml

```diff
@@ -71,12 +71,11 @@ timeseries_analyzer:
   max_days: 3600
 
 anovos_basic_report:
-  basic_report: True
+  basic_report: False
```

Comment:
This is the basic report config. Is there any reason why `basic_report` is set to False?

Reply:
Yes. The basic_report triggers the computation of a correlation matrix, which leads to a memory overflow. The version that is available on PyPI does not yet have the `skip_corr_matrix` option that was introduced to solve this exact issue. Hence, I had to turn off the report generation for the workflow to run through.

Once a new version of Anovos is released, it will have that option and we can run the basic_configs.yaml workflow as it is currently available on the main branch.

Reply:
Then I would suggest us to do this as a separated PR to better document it, and then after Anovos got another release, we can revert the PR and bring it back all at once.

Reply:
I don't follow. The issue has been fixed way back in May by introducing the `skip_corr_matrix` flag (https://github.com/anovos/anovos/commit/e0da774cd1cf0a7c4e9d0fda5c78f711e1bceb3f) and setting it to `True` everywhere (https://github.com/anovos/anovos/commit/cfeb76cd1498868219080f137514a224a02bb7f5).

Unfortunately, there hasn't been a new release since then, so the latest release that is available on PyPI does not support this flag. Hence, in order to run the workflow with this release, we need to omit the report generation.

Reply:
Got it. So I would take it that, after this PR is merged and we release 0.4.0 on PyPI, we then can open a new PR to change this back to `True` and bring back all the `skip_corr_matrix` and `report_generation` in configs_basic.yaml right?

Reply:
Either that, or we wait, or live with the fact that the local_execution workflow is broken on `main`. I'd prefer to wait until right before the release but ultimately it doesn't really matter. Your choice :-)

Reply:
I think it's better to wait right before the release as well. @kajanansangar FYI